hist/hist/src/TFormula.cxx

```diff
@@ -1670,55 +1672,57 @@ void TFormula::HandleExponentiation(TString &formula)
    }
 }
 
+namespace {
+
+// Borrowed from RooHelpers.h, should this go to a central place?
```

Comment:
I agree, this should totally go into `core/foundation/inc/ROOT/`. Should this return `vector<string>` or `vector<string_view>`? The latter is dangerous if people do `tokenize("abc cde",...)` but I find that unlikely to happen.

Reply:
Okay! I don't have time to do this now, but I'll open a GitHub issue about it. Maybe we should have two versions: one returning `vector<string>` and one returning `vector<string_view>`.